### DISCUSSION TITLE
Allow Listener to Skip Song

### DISCUSSION THREAD

**Discussion by countrymusicfy** [AzuraCast/AzuraCast#4672] [Q&A] [answered]
Is there a feature that allows a listener to skip a song similar to what I see in the admin dashboard?

**Comment by Vaalyn** [comment, marked as answer]
No there is currently no skip function available for listeners.

There is already a feature request for this though: https://features.azuracast.com/suggestions/217034/add-voteskip-button

Due to an issue/bug on LS the skip function itself is a bit broken in the current version leading it to stray of the queue of the AzuraCast autodj. This will be fixed in the update for Liquidsoap 2.0 though and when that is live something like a vote-skip feature could theoretically be done.

**Reply by ryanginnelly252** [reply]
Seems interesting! would be cool to see this in action soon ;)

**Comment by bugoverfl0w** [comment]
on frontend you request to your backend, and backend you request to api to skip song
